Examples/SendValuesDeviceType/SendValuesDeviceType.cpp

```diff
@@ -0,0 +1,57 @@
+// This example sends data to multiple variables to
+// Ubidots through HTTP protocol, and uses Device Types feature
+
+// IMPORTANT: Device type are only supported through HTTP
+
+/****************************************
+ * Include Libraries
+ ****************************************/
+
+#include "Ubidots.h"
+
+/****************************************
+ * Define Instances and Constants
+ ****************************************/
+
+const char* device_label = "my-device";  // Edit here your device label
```

Comment:
The device label should be the device's MAC. This ought to be defined by the library.